emgapianns/management/commands/import_assembly.py

```diff
@@ -81,29 +81,16 @@ def get_ena_db_assembly(self, accession):
 
     def create_or_update_assembly(self, ena_assembly):
         accession = ena_assembly.assembly_id
-
         logger.info("Creating assembly {}".format(accession))
-        status = emg_models.Status.objects.using(self.emg_db).get(status_id=ena_assembly.status_id)
         defaults = sanitise_fields({
-            "status_id": status,
+            "is_private": ena_assembly.status_id == 2,
+            "is_suppressed": ena_assembly.status_id in [3, 5 ,6],
             "wgs_accession": ena_assembly.wgs_accession,
             "legacy_accession": ena_assembly.gc_id,
             "coverage": ena_assembly.coverage,
             "min_gap_length": ena_assembly.min_gap_length
         })
-        assembly, created = emg_models.Assembly.objects.using(self.emg_db).update_or_create(
-            accession=accession,
-            defaults=defaults
-        )
-        return assembly
-
-    def update_assembly_status(self, accession, status):
-
-        logger.info("Updating assembly {} to {}".format(accession, status))
-        status = emg_models.Status.objects.using(self.emg_db).get(status=status)
-        defaults = sanitise_fields({
-            "status_id": status,
-        })
+        print(defaults)
```

Comment:
```suggestion
```
debugging leftover? or switch to `logging.info`

Reply:
ups